draft-ietf-tls-tls13.md

```diff
@@ -1566,7 +1566,9 @@ cipher_suites
   order of client preference. If the list contains cipher suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
-  usual. Values are defined in {{cipher-suites}}.
+  usual. Values are defined in {{cipher-suites}}. If the client is
+  attempting a PSK key establishment, it MUST only advertise the single
+  cipher suite associated with the PSK.
```

Comment:
This actually isn't correct. You MUST advertise it, but it's not right to require that you MUST only advertise it because this makes it impossible for servers to disable any cipher while a ticket is still valid because it might be in use by some client.